combustion-android-ble/src/main/java/inc/combustion/framework/ble/ProbeManager.kt

```diff
@@ -148,7 +148,14 @@ internal open class ProbeManager (
         addJob(owner.lifecycleScope.launch(Dispatchers.IO) {
             while(isActive) {
                 if(isConnected.get() && mac != SimulatedProbeManager.SIMULATED_MAC) {
-                    remoteRssi.set(peripheral.rssi())
+                    try {
+                        remoteRssi.set(peripheral.rssi())
+                    } catch (e: Exception) {
+                        if (DebugSettings.DEBUG_LOG_BLE_OPERATIONS) {
+                            Log.d(LOG_TAG,
```

Comment:
What kind of exceptions were you seeing here? Is `debug` the right level of logging? Is there one that we can explicitly ignore? If so, it might make sense to catch that type of exception rather than the generic `Exception`.